### PR TITLE
Inline all aws dashboards

### PR DESCRIPTION
**Pull request by flash1293**
## What does this PR do?

Inlines all visualizations into their respective dashboards and deletes the individual visualization saved objects using https://github.com/flash1293/legacy_vis_analyzer

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
~- [ ] I have verified that all data streams collect metrics or logs.~
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] All panels on all dashboards still work and show the right data

## Related issues

Fixes https://github.com/elastic/integrations/issues/3454

### CODE REVIEW

File: packages/aws/manifest.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ categories:
   - cloud
 release: ga
 conditions:
-  kibana.version: "^7.15.0 || ^8.0.0"
+  kibana.version: "^8.1.0"
```
Comment (andresrc) [inline packages/aws/manifest.yml:13]: Before merging we should see if these changes work in 7.17.x

Comment (flash1293) [reply packages/aws/manifest.yml:13]: As mentioned via email, as the aws dashboards are partially very old the inliner script only works for version 8.1 and up

Comment (kaiyan-sheng) [reply packages/aws/manifest.yml:13]: I tried to install this integration(changing the kibana.version to include 7.17.0 and above) with running 7.17.5 stack version and here is the error I got: 
```
Error installing aws 1.18.1: Document "aws-0eb5a6a0-694f-11ea-b0ac-95d4ecb1fecd" has property "dashboard" which belongs to a more recent version of Kibana [8.1.0]. The last known version is [7.17.3]
```

Comment (flash1293) [reply packages/aws/manifest.yml:13]: Correct, they only work for 8.1 and above - 7.x or 8.0 is not supported

**Review by kaiyan-sheng — APPROVED** [review]
I tested some of the dashboards with this change and the dashboards do load. But some of them don't show data caused by the known bug in the dashboards in https://github.com/elastic/integrations/issues/3703.

**Review by andrewctate — APPROVED** [review]
Looked at a random sample of five dashboards and all panels were by-value. Great work!

**Comment by flash1293** [comment]
@kaiyan-sheng could you take another look? Had to re-inline due to merge conflicts

**Review by kaiyan-sheng — APPROVED** [review]

**Pull request merged**